sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/jdbc/connection/ConnectionProviderSuite.scala

```diff
@@ -17,12 +17,21 @@
 
 package org.apache.spark.sql.execution.datasources.jdbc.connection
 
+import java.sql.{Connection, Driver}
 import javax.security.auth.login.Configuration
 
+import org.mockito.Mockito._
```

Comment:
shall we remove this import:

```
[error] /home/runner/work/spark/spark/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/jdbc/connection/ConnectionProviderSuite.scala:23:28: Unused import
[error] import org.mockito.Mockito._
[error]                            ^
[error] one error found
[error] (sql / Test / compileIncremental) Compilation failed
[error] Total time: 599 s (09:59), completed Jul 15, 2021 2:48:41 PM
```

Reply:
Yes, I will redesign the tests to work around this issue.